apps/fm/js/fm.js

```diff
@@ -918,11 +918,18 @@ function init() {
   );
 }
 
-window.addEventListener('load', function(e) {
-  AirplaneModeHelper.ready(function() {
-    airplaneModeEnabled = AirplaneModeHelper.getStatus() == 'enabled';
-    init();
-
+navigator.mozL10n.once(function(e) {
```

Comment:
but if it's loaded right after l10n.js, then it's okay because the l10n logic is run asynchronously.